lib/sumsub/request.rb

```diff
@@ -184,6 +186,7 @@ def build_header(
         resource: resource, 
         body: body, 
         method: method,
+        secret_key: @secret_key,
```

Comment:
I think there's no need for this. `MessageSigner` is already receiving `secret_key` from `Sumsub.configuration.secret_key`.

Reply:
@rwehresmann  `@secret_key` from `#initialize` is not used in this class.

Reply:
Well, I'm thinking and you may be right. It's interesting in case you wanna instantiate `Sumsub::Request` informing another credential than the one informed in `Sumsub::Configuration`. I can imagine myself in a scenario where I would like to do that in ruby console, for instance. I'll considered it, thanks.